arm-network/2017-06-01/swagger/networkSecurityGroup.json

```diff
@@ -468,9 +469,23 @@
           "type": "string",
           "description": "The CIDR or source IP range. Asterix '*' can also be used to match all source IPs. Default tags such as 'VirtualNetwork', 'AzureLoadBalancer' and 'Internet' can also be used. If this is an ingress rule, specifies where network traffic originates from. "
         },
+        "sourceAddressPrefixes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The CIDR or source IP ranges."
```

Comment:
@avijitgupta What would you like this description to be, based on Sandhya's feedback?